src/ProgramBlockEditor.test.js

```diff
@@ -109,7 +109,7 @@ function createMountProgramBlockEditor(props) {
 }
 
 function confirmDeleteAllModalIsOpen(programBlockEditorWrapper): boolean {
-    return programBlockEditorWrapper.exists('.ConfirmDeleteAllModal');
+    return programBlockEditorWrapper.exists('.Modal__container.active');
```

Comment:
This checks that a modal has been opened. Not that the Delete All modal specifically has been opened.